parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -336,7 +336,7 @@ After=bind-mount.service
 EOF
 fi
 
-logs_to_events "AKS.CSE.ensureSysctl" ensureSysctl
+logs_to_events "AKS.CSE.ensureSysctl" ensureSysctl || exit $ERR_SYSCTL_RELOAD
```

Comment:
qq: does ensureSysctl itself not have an exit code? wondering if we should stick to a pattern of 

ensureSomething || exit $?

as we did with enableLocalDNS

just a thought, not a hard block, but will help keep code uniform.

Reply:
It doesn't at the moment, ir uses `retrycmd_if_failure` and will return 1 if that fails, but no specific code there from what I can see.